test/starlark_tests/targets_under_test/ios/BUILD

```diff
@@ -1026,29 +1025,21 @@ ios_extension(
 
 # ---------------------------------------------------------------------------------------
 
-substitution(
-    name = "resource_rules",
-    src = "//test/starlark_tests/resources:ResourceRules.plist",
-    tags = FIXTURE_TAGS,
-    var_name = "RULES",
-)
```

Comment:
we should def keep this test for now

Reply:
which one? the test is still there, only that toolchain rule is removed (since it's not useful anymore)

Reply:
or perhaps you're advising we keep both behaviours ?

Reply:
otherwise this is a breaking change right?

Reply:
which we could do ofc, but im not sure we want to?

Reply:
it would be a breaking change indeed. knowing how difficult it was to implement, I figured those who were using it would welcome the change anyway (since it's so much simpler)